### DISCUSSION TITLE
Filter out or extract entries by tags?

### DISCUSSION THREAD

**Discussion by brownLlama** [nvim-neorg/neorg#1284] [General]
Hi team, I'm new to Neorg and neovim, with that said, I'm using Neorg as my daily driver for my note taking. I normally, create a new page per day as journal and I like to write everything. For example my todos, different tasks for different projects and so on. I always tag them with their respective titles. Let's assume that I created a list of todo yesterday and last week with the tag "my_todos". Now if I like to list out / filter out / extract all of "my_todos" in a single page/preview, is that possible? So that I know, which of my TODO are done and which need to be completed? Or is there any alternative for it?? Thanks.

**Comment by Anrock** [comment]
Sounds like GTD agenda which currently in the works. Stay tuned.

**Reply by brownLlama** [reply]
nice!! Thanks. 🙏🏽
